From f3cecb7fef095d0ce6fb34c40882f64dbb863074 Mon Sep 17 00:00:00 2001
From: thoudayer <thoudayer@ludia.com>
Date: Fri, 14 Jun 2013 11:23:56 -0400
Subject: [PATCH] In file 'AbstractBuild/index.jelly', stop to try to access
 fingerprint of artifacts because some fingerprints are from
 another project that some users don't have access!

---
 .../hudson/model/AbstractBuild/index.jelly         |   32 --------------------
 1 files changed, 0 insertions(+), 32 deletions(-)

diff --git a/core/src/main/resources/hudson/model/AbstractBuild/index.jelly b/core/src/main/resources/hudson/model/AbstractBuild/index.jelly
index 1879ad1..f3b97f9 100644
--- a/core/src/main/resources/hudson/model/AbstractBuild/index.jelly
+++ b/core/src/main/resources/hudson/model/AbstractBuild/index.jelly
@@ -65,38 +65,6 @@ THE SOFTWARE.
           <j:choose>
             <j:when test="${it.hasChangeSetComputed()}">
               <st:include it="${set}" page="digest.jelly" />
-
-              <!-- dependency changes -->
-              <j:set var="depChanges" value="${it.getDependencyChanges(it.previousBuild)}"/>
-              <j:if test="${!empty(depChanges)}">
-                ${%Changes in dependency}
-                <ol>
-                  <j:forEach var="dep" items="${depChanges.values()}">
-                    <li>
-                      <a href="${rootURL}/${dep.project.url}" class="model-link">${dep.project.displayName}</a>
-                      <st:nbsp/>
-                      <j:choose>
-                        <j:when test="${dep.from!=null}">
-                          <a href="${rootURL}/${dep.from.url}" class="model-link inside">
-                            <img src="${imagesURL}/16x16/${dep.from.buildStatusUrl}"
-                                 alt="${dep.from.iconColor.description}" height="16" width="16" />${dep.from.displayName}</a>
-                        </j:when>
-                        <j:otherwise>
-                          ?
-                        </j:otherwise>
-                      </j:choose>
-
-                      &#x2192; <!-- right arrow -->
-
-                      <a href="${rootURL}/${dep.to.url}" class="model-link inside">
-                        <img src="${imagesURL}/16x16/${dep.to.buildStatusUrl}"
-                             alt="${dep.to.iconColor.description}" height="16" width="16" />${dep.to.displayName}</a>
-
-                      (<a href="${rootURL}/${dep.project.url}changes?from=${dep.fromId}&amp;to=${dep.toId}">${%detail}</a>)
-                    </li>
-                  </j:forEach>
-                </ol>
-              </j:if>
             </j:when>
             <j:when test="${it.building}">
               ${%Not yet determined}
-- 
1.7.9

